test/testutils.jl

```diff
@@ -26,7 +26,7 @@ end
 # New Julia implementation
 function juliaprojjsonstring(code; multiline=false)
   wkt2str = CoordRefSystems.wkt2(code)
-  wkt2toprojjson(wkt2str, multiline=multiline)
+  GeoIO.wkt2toprojjson(wkt2str, multiline=multiline)
 end
```

Comment:
I believe this function is not necessary, assuming that the new `GeoIO.projjsonstring` is already in place. You just need the old definition in this file for comparison purposes.